src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.sendletter.controllers;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.sendletter.model.Letter;
+import uk.gov.hmcts.reform.sendletter.services.LetterService;
+
+import java.util.Map;
+
+@RestController
+@RequestMapping(
+    path = "letter",
```

Comment:
`/letters`
https://hmcts.github.io/restful-api-standards/#134

Reply:
Thanks Rafal.Will update this :)